frontend/canvas-chrome-ext/src/manifest.json

```diff
@@ -8,7 +8,10 @@
   },
 
   "web_accessible_resources": [{
-    "resources": ["scripts/inject_script.js", "components/compilation_notification.html", "scripts/notification.js"],
+    "resources": ["scripts/inject_script.js",
```

Comment:
just fyi, in a future PR we're gonna rename this something like "StudentInjectScript" to differentiate between inject scripts for the instructor and student side, since we're adding another inject script for the instructor side